.github/workflows/tests.yml

```diff
@@ -571,6 +571,7 @@ jobs:
         env:
           POSTGRES: ${{ (matrix.database == 'Postgres') && 1 || '' }}
           WORKERS: ${{ (matrix.arrangement == 'workers') && 1 || '' }}
+          UNIX_SOCKETS: 1
         name: Run Complement Tests
 
   cargo-test:
```

Comment:
Can you give an example config in the issue description on how one would utilize this? I know you have instructions on how to compose things on top of what https://github.com/matrix-org/synapse/pull/15353 says but seems a bit fiddly and would be easier to reference as a whole.

Reply:
Well, I guess I could make a complete example of before and after. I tried to do that in the docs, but here for reference, sure.

That being said, this spot in the code is for Complement to run on Unix sockets(which I should probably document as well, so thanks for that).

Reply:
Almost forgot, added how to use this in [86c5fd5](https://github.com/matrix-org/synapse/pull/15708/commits/86c5fd5ed98dbeec3f3ffde7b481a513b9d24e09)